pkg/R/utils.R

```diff
@@ -305,11 +305,27 @@ processClosure <- function(node, oldEnv, argNames, newEnv) {
               !(nodeChar %in% getNamespaceExports("SparkR")))) {  # Only include SparkR internals.
           # Set parameter 'inherits' to FALSE since we do not need to search in
           # attached package environments.
-          if (exists(nodeChar, envir = func.env, inherits = FALSE)) {
+          if (tryCatch(exists(nodeChar, envir = func.env, inherits = FALSE),
```

Comment:
At least one error I came across was if the `nodeChar` is `""`, the `get` return a error of `invalid argument`. So I thought that should be a `FALSE` in any case of error.